subscription/upgrade.md

```diff
@@ -17,28 +17,21 @@ The following section contains information on how to purchase a Docker  subscrip
 
 To purchase a Docker subscription:
 
-1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
+1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account and click **Upgrade**.
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing** to go to the **Billing Details** page. This page displays billing information about your existing user account.
+2. Select your subscription and click **Buy Now**.
 
    > **Note**
    >
-   > Ensure you have selected the correct account on the left side of the
-   **Billing Details** page. For example, if you want to purchase a Docker
-   Pro subscription, you must select your personal account.
+   > If you do not have a subscription, navigate to Organizations and click **Create  Organization**. Proceed with the on screen instructions and click **Continue to Payment**.
```

Comment:
I think you mean `if you do not have an organization` 🤔  ? Although the context is a bit confusing, we switched from a personal to a team subscription. 

Maybe we should have 2 sections `purchase a personal subscription` and `purchase a team subscription`

Reply:
Thanks! @amineck  I updated organization...Also, for Personal subscription, isn't it free?  The next section below is to upgrade your Pro, Team or Business subscriptions